### ISSUE TITLE
ZAT Tools not working on macOS Catalina

### ISSUE THREAD

**Issue by verschoren** [zendesk/zendesk_apps_tools#318]
The macOS Catalina update moves Ruby to v2.6 so it currently refuses to run the ZAT Tools.

It throws the following error:
`premiumusthomas:~ thomasverschoren$ zat new
-bash: /usr/local/bin/zat: /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/bin/ruby: bad interpreter: No such file or directory`

**Comment by peacefulseeker** [comment]
@verschoren , did you solve the issue somehow?

**Comment by verschoren** [comment]
Yep:
[see here](https://verschoren.com/2019/10/zat-tools-catalina)

**Comment by peacefulseeker** [comment]
Yeah saw, it. 👍 
But seems I have some troubles with developer tools. 
After running
![image](https://user-images.githubusercontent.com/25010454/70509433-a0042980-1b37-11ea-8ded-0eacd9f26049.png)
And eventually:
![image](https://user-images.githubusercontent.com/25010454/70509326-97abee80-1b37-11ea-9da8-64414b164686.png)

Tried to reinstall it, but did not work either. Doing something wrong. 
Do you have any ideas? :)

**Comment by verschoren** [comment]
Two ideas:

1. Try running `brew config` it will probably ask you to accept the XCode terms and conditions. 
2. Force a full update with `softwareupdate --install -a`

**Comment by peacefulseeker** [comment]
> Two ideas:
> 
> 1. Try running `brew config` it will probably ask you to accept the XCode terms and conditions.
> 2. Force a full update with `softwareupdate --install -a`

1) Did not work unfortunately:
![image](https://user-images.githubusercontent.com/25010454/70802865-5166c180-1dbb-11ea-8c9f-634fd0f44ebf.png)
2) `No updates are available.`

Perhaps this article could be useful for somebody as well.
Tried to use local ruby and gem, but without success again.
https://medium.com/faun/macos-catalina-xcode-homebrew-gems-developer-headaches-cf7b1edf10b7
Will inform here what worked in my case and what I did wrong.
